frontend/public/module/k8s/cluster-settings.ts

```diff
@@ -131,14 +131,48 @@ export const getOpenShiftVersion = (cv: ClusterVersionKind): string => {
   return lastUpdate.state === 'Partial' ? `Updating to ${lastUpdate.version}` : lastUpdate.version;
 };
 
-// example link: https://access.redhat.com/downloads/content/290/ver=4.1/rhel---7/4.1.13/x86_64/product-errata
-export const getErrataLink = (cv: ClusterVersionKind): string => {
+type ParsedVersion = {
+  major: number;
+  minor: number;
+  patch: number;
+  prerelease: string[];
+};
+
+const getVersion = (cv: ClusterVersionKind): ParsedVersion => {
   const version: string = _.get(cv, 'status.history[0].version');
   if (!version) {
     return null;
   }
 
-  const parsed = semver.parse(version);
+  return semver.parse(version);
+};
+
+export const getReportBugLink = (cv: ClusterVersionKind): { label: string; href: string } => {
+  const parsed = getVersion(cv);
+  if (!parsed) {
+    return null;
+  }
+
+  // Show a Bugzilla link for prerelease versions and a support case link for supported versions.
+  const { major, minor, patch, prerelease } = parsed;
+  return _.isEmpty(prerelease)
+    ? {
+        label: 'Open Support Case',
+        href: `https://access.redhat.com/support/cases/#/case/new?product=OpenShift%20Container%20Platform&version=${major}.${minor}&clusterId=${
+          cv.spec.clusterID
+        }`,
+      }
+    : {
+        label: 'Report Bug',
+        href: `https://bugzilla.redhat.com/enter_bug.cgi?product=OpenShift%20Container%20Platform&version=${major}.${minor}.${
```

Comment:
If we can feed a description here I think we should include the cluster id.

Reply:
I've found a way to prefill both environment and description fields. I'm leaning toward adding cluster ID to environment so we don't overwrite the template description.

Reply:
I like this as an idea it also gives cee a location to save cluster ids